distributed/cli/utils.py

```diff
@@ -26,3 +28,32 @@ def handle_signal(signum, frame):
         old_handlers[sig] = signal.signal(sig, handle_signal)
 
     await event.wait()
+
+
+def install_signal_handlers(loop=None, cleanup=None):
+    """
+    Install global signal handlers to halt the Tornado IOLoop in case of
+    a SIGINT or SIGTERM.  *cleanup* is an optional callback called,
+    before the loop stops, with a single signal number argument.
+    """
+    import signal
```

Comment:
I think we should move `async def wait_for_signals` into a private `distributed._signals` module and deprecate the `distributed.cli.utils` module

Reply:
I like that!

Reply:
distributed itself is not needing this, is it? We should probably raise a DeprecationWarning and remove this code in a later release or is this something worth keeping around? If so, we should add tests but I'm not very fond of keeping this around, to be honest.

Reply:
FWIW I'm fine merging as is to unblock upstream CI but we should settle the question about deprecation asap

Reply:
See #6367 for the follow-up PR that cleans things up.